charts/kube-prometheus-stack/hack/sync_prometheus_rules.py

```diff
@@ -105,6 +112,8 @@ def new_representer(dumper, data):
     'config-reloaders': ' .Values.defaultRules.rules.configReloaders',
     'etcd': ' .Values.kubeEtcd.enabled .Values.defaultRules.rules.etcd',
     'general.rules': ' .Values.defaultRules.rules.general',
+    'grafana_rules': ' .Values.defaultRules.rules.grafana',
+    'GrafanaAlerts': ' .Values.defaultRules.rules.grafana',
```

Comment:
Two questions:
- are these supposed to be the same?
- can we rename `GrafanaAlerts` to `grafana-alerts` to make it similar with the other names and filenames?

Reply:
> can we rename GrafanaAlerts to grafana-alerts to make it similar with the other names and filenames?

I saw this, too. Currently, the file name based on the rule group name, which behave differently at Grafana. But i can add a replacement for this, like to snake case.

> are these supposed to be the same?

GrafanaAlerts has an dependency against grafana_rules. And there are just 2 rules in total. I thought, its fine to group them. But I'm also open separate them.

No strong opinion here.